src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForSimulation.cs

```diff
@@ -94,6 +97,11 @@ public IBuildingBlock TemplateBuildingBlockFor(IBuildingBlock buildingBlock)
          return _templateResolverTask.TemplateBuildingBlockFor(buildingBlock);
       }
 
+      public Module TemplateModuleFor(Module module)
```

Comment:
let's try to inline those now. I know it's a matter of preference ultimately but I'd like to simplify those simple one liner